controllers/work_controller_test.go

```diff
@@ -104,6 +104,10 @@ var _ = Context("WorkReconciler.Reconcile()", func() {
 			workPlacementGenID = workPlacementList.Items[0].GetGeneration()
 		})
 
+		AfterEach(func() {
+			deleteWork(work)
+		})
+
```

Comment:
this `AfterEach` block will run after the `It` in the `When` block below. Moving it to here sounds... odd? i'd be fine moving it up (just after the `BeforeEach`) or as the last thing (as it was before) 🤔

Reply:
I think @abangser moved this before I started working on branch (or I have no memory of this change). Happy to move it back